### ISSUE TITLE
Error: pandoc document conversion failed with error 1

### ISSUE THREAD

**Issue by BonnieGu** [rstudio/rmarkdown#956]
processing file: hw01_cg9.Rmd
  |..                                                               |   3%
  ordinary text without R code

  |....                                                             |   6%
label: unnamed-chunk-1 (with options) 
List of 2
 $ message: logi FALSE
 $ warning: logi FALSE

  |......                                                           |   9%
  ordinary text without R code

  |.......                                                          |  11%
label: unnamed-chunk-2
  |.........                                                        |  14%
  ordinary text without R code

  |...........                                                      |  17%
label: unnamed-chunk-3
Parsed with column specification:
cols(
  y = col_double(),
  a = col_double(),
  b = col_double(),
  c = col_double(),
  d = col_integer()
)
  |.............                                                    |  20%
  ordinary text without R code

  |...............                                                  |  23%
label: unnamed-chunk-4
  |.................                                                |  26%
  ordinary text without R code

  |...................                                              |  29%
label: unnamed-chunk-5
  |....................                                             |  31%
  ordinary text without R code

  |......................                                           |  34%
label: unnamed-chunk-6
  |........................                                         |  37%
   inline R code fragments

  |..........................                                       |  40%
label: unnamed-chunk-7 (with options) 
List of 2
 $ message: logi TRUE
 $ warning: logi TRUE

  |............................                                     |  43%
  ordinary text without R code

  |..............................                                   |  46%
label: unnamed-chunk-8
  |................................                                 |  49%
   inline R code fragments

  |.................................                                |  51%
label: unnamed-chunk-9 (with options) 
List of 2
 $ message: logi TRUE
 $ warning: logi TRUE

  |...................................                              |  54%
  ordinary text without R code

  |.....................................                            |  57%
label: unnamed-chunk-10
  |.......................................                          |  60%
  ordinary text without R code

  |.........................................                        |  63%
label: unnamed-chunk-11
  |...........................................                      |  66%
  ordinary text without R code

  |.............................................                    |  69%
label: unnamed-chunk-12
  |..............................................                   |  71%
  ordinary text without R code

  |................................................                 |  74%
label: unnamed-chunk-13
  |..................................................               |  77%
   inline R code fragments

  |....................................................             |  80%
label: unnamed-chunk-14
  |......................................................           |  83%
  ordinary text without R code

  |........................................................         |  86%
label: unnamed-chunk-15
  |..........................................................       |  89%
  ordinary text without R code

  |...........................................................      |  91%
label: unnamed-chunk-16
  |.............................................................    |  94%
   inline R code fragments

  |...............................................................  |  97%
label: unnamed-chunk-17
  |.................................................................| 100%
  ordinary text without R code

/Applications/RStudio.app/Contents/MacOS/pandoc/pandoc +RTS -K512m -RTS hw01_cg9.utf8.md --to html --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash --output hw01_cg9.html --smart --email-obfuscation none --self-contained --standalone --section-divs --template /Library/Frameworks/R.framework/Versions/3.3/Resources/library/rmarkdown/rmd/h/default.html --no-highlight --variable highlightjs=1 --variable 'theme:bootstrap' --include-in-header /var/folders/wq/ljbdffhn3mb7w5vlr054dh4m0000gn/T//RtmpaQGFXb/rmarkdown-str2d67475dfc5.html --mathjax --variable 'mathjax-url:https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' 
output file: hw01_cg9.knit.md

pandoc: hw01_cg9.html: openFile: permission denied (Permission denied)
Error: pandoc document conversion failed with error 1
Execution halted

Yesterday, I could knit the .rmd file normally. Today, I just add some notes on the file, but there is something wrong with it.
Anybody meets the same problem? How can I solve this problem?

**Comment by Struggle-YD** [comment]
I had the same trouble

**Comment by yihui** [comment]
These issues are probably similar: https://github.com/yihui/knitr/issues/1248 https://github.com/yihui/knitr/issues/1209 but they only occurred on Windows. I have no idea how it could happen on macOS.

**Comment by yihui** [comment]
Sorry, but I don't think we will ever be able to solve this problem without being able to reproduce it. The only advice I could give is to [upgrade R, RStudio, and all your R packages](https://yihui.name/en/2017/05/when-in-doubt-upgrade/), and see if the problem persists.
